app/presenters/queries/content_item_presenter.rb

```diff
@@ -125,6 +128,19 @@ def publication_state_sql
         SQL
       end
 
+      def state_history_sql
+        <<-SQL
+          (
+            SELECT json_agg((number, name))
+            FROM content_items c
```

Comment:
In this case, an alias is needed to disambiguate the joined table in this restriction:
`WHERE c.content_id = content_items.content_id`

Happy to change the `s` and `u` aliases if you think, though?